stack/setup/main.tf

```diff
@@ -60,6 +60,8 @@ module "secrets-kms-key" {
 }
 
 resource "aws_acm_certificate" "main" {
+  count = var.is_read_replica_on_same_domain ? 0 : 1
```

Comment:
Do you mean the use case? 
It comes from this PR -> https://github.com/dbl-works/terraform/pull/97

And the reason I remove the creation of AWS Cert is because
-> If it is the same domain, we can reuse the same cert and don't need to create a new one.